src/main/kotlin/frc/robot/subsystems/drive/Drive.java

```diff
@@ -510,4 +512,21 @@ public void setVoltage(@NotNull Voltage voltage) {
             return null;
         };
     }
+
+    public void followTrajectory(SwerveSample sample) {
+        // Get the current pose of the robot
+        Pose2d pose = getPose();
+
+        // Generate the next speeds for the robot
+        ChassisSpeeds speeds =
+                new ChassisSpeeds(
+                        sample.vx + TunerConstants.xController.calculate(pose.getX(), sample.x),
+                        sample.vy + TunerConstants.yController.calculate(pose.getY(), sample.y),
+                        sample.omega
+                                + TunerConstants.headingController.calculate(
+                                        pose.getRotation().getRadians(), sample.heading));
+
+        InitializerKt.getDrive()
+                .runVelocity(ChassisSpeeds.fromFieldRelativeSpeeds(speeds, getRotation()));
```

Comment:
If `getDrive()` returns `Drive` you can just do this:
```suggestion
        runVelocity(ChassisSpeeds.fromFieldRelativeSpeeds(speeds, getRotation()));
```
Because you're inside this class.